helm_deploy/hmpps-book-secure-move-frontend/templates/_envs.tpl

```diff
@@ -0,0 +1,116 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Environment variables for web and worker containers
+*/}}
+{{- define "deployment.envs" }}
+env:
+  - name: API_CLIENT_ID_KEY
```

Comment:
If this is where we're setting environment variables, then `.docs/environment-variables.md` should also be updated with guidelines to update here as well.

Reply:
I'll go over the docs and update ones I've confirmed what's here works. There are a couple of things I'd like to see working in a stage deploy before updating the docs. But I will definitely be doing that as part of updating the other environments.

Reply:
Will this supersede what we currently have in the deployment repos?

Reply:
Where it's deployed by helm, yes. I'll clean up the deployment repos to reflect this when it's working for sure